src/query/typed_value.hpp

```diff
@@ -96,6 +97,7 @@ class TypedValue {
   using TString = utils::pmr::string;
   using TVector = utils::pmr::vector<TypedValue>;
   using TMap = utils::pmr::map<utils::pmr::string, TypedValue>;
+  using TSet = utils::pmr::unordered_set<TypedValue, TypedValue::Hash, TypedValue::BoolEqual>;
```

Comment:
I think this type alias is not in a good place here. All of the above containers type are defined here because can be contained by a `TypedValue`, but this `TSet` is not, thus it is confusing. I think it would be better to define this alias in `AggregateCursor::AggregationValue`.